oio/xcute/common/backend.py

```diff
@@ -558,7 +558,7 @@ def run_next(self, orchestrator_id):
         job_info = self.script_run_next(
             keys=[self._get_timestamp(), orchestrator_id],
             client=self.conn)
-        if job_info is None:
+        if not job_info:
```

Comment:
is it covering another case?

Reply:
No, it isn't covering another case.
To avoid reproducing this error (https://github.com/open-io/oio-sds/pull/1945/files#diff-4a7c22a19c01798de356ba370cc8586bL632), I preferred to check `job_info` always in the same way.